app/models/user.rb

```diff
@@ -263,13 +263,19 @@ def self.from_omniauth(auth)
   #
   # @return [JSON] with symbols as keys
   def get_preferences(key)
-    if self.pref.present? && self.pref.settings[key.to_s].present?
-      return self.pref.settings[key.to_s].deep_symbolize_keys
-    elsif Pref.default_settings
-      return Pref.default_settings[key.to_sym] || Pref.default_settings[key.to_s]
-    else
-      return nil
+    existing = self.pref.settings[key.to_s].deep_symbolize_keys
```

Comment:
Refactored this because it does not allow for new email preferences to be added to the defaults. The hash returned to the controller is used to build the page. The hash looks like: `{'users': {'key1':'val1', 'key2':'val2'},'admins': ...}`. The check was always just pulling the data from the DB and ignoring new prefs for the 'users' key. 
Instead updated to add new defaults to the existing prefs. We would need to refactor if we ever allow deeper nesting within this hash